inspirehep/modules/records/validators/rules.py

```diff
@@ -0,0 +1,156 @@
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014, 2015, 2016, 2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+'''Validator classes for Invenio Record Editor.'''
+
+from __future__ import absolute_import, division, print_function
+
+from .helpers import (
+    check_if_field_exists_in_dict_list_x_times,
+    check_field_values_not_in_required_values_for_record,
+    check_if_listfield_property_values_are_valid,
+    query_db_for_duplicates_for_field_listitem_with_index,
+    FIELD_REQUIRE_FIELD_VALUES_TEMPLATE,
+    FIELD_VALIDATION_TEMPLATE,
+    FIELD_REQUIRE_FIELD_TEMPLATE
+)
+from jsonschema import ValidationError
+from idutils import is_isbn
+import pycountry
+
+
+'''
+Error Validator Functions
+'''
```

Comment:
That said, this file does not even have sections, so what is this lone docstring?